src/java/org/apache/cassandra/service/StorageService.java

```diff
@@ -921,7 +934,7 @@ public boolean isReplacing()
         if (replacing)
             return true;
 
-        if (System.getProperty("cassandra.replace_address_first_boot", null) != null && SystemKeyspace.bootstrapComplete())
+        if (REPLACE_ADDRESS_FIRST_BOOT.getString(null) != null && SystemKeyspace.bootstrapComplete())
```

Comment:
getString(null) is redundant, getString() is enough.

Reply:
Fixed.